### PR TITLE
my activity

### PR DESCRIPTION
**Pull request by taksd**
test

### CODE REVIEW

File: recommendations/taksd.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Japan
```
Comment (yi200001) [inline recommendations/taksd.md:1]: Yoichi comment

Comment (taksd) [inline recommendations/taksd.md:1]: Got it by Tak

**Comment by yi200001** [comment]
This looks good

**Comment by taksd** [comment]
Merged without problem. Tak

**Pull request merged**